flowback/group/services.py

```diff
@@ -516,11 +516,7 @@ def group_thread_delete(user_id: int, thread_id: int):
     thread.delete()
 
 
-def group_thread_comment_create(author_id: int,
-                                thread_id: int,
-                                message: str = None,
-                                attachments: list = None,
-                                parent_id: int = None):
+def group_thread_comment_create(*,author_id, thread_id, parent_id, message,attachments,**kwargs):
```

Comment:
It'd be better having proper typehints to help any future maintainers of code, including yourself

Additionally people should be able to send either message or attachments

Also kwargs have no place in here

Reply:
Updated to have type hints and remove the kwargs